include/wil/registry_helpers.h

```diff
@@ -182,7 +183,7 @@ namespace wil
                 if (last - first < 3)
                 {
                     // it doesn't have the required 2 terminating null characters - return an empty string
-                    return { ::std::wstring{} };
+                    return ::std::vector<::std::wstring>(1);

```

Comment:
NOTE: I kept the behavior here as-is, but it kind of seems odd to return a vector with a single empty element. Should this not just be an empty vector?

Reply:
*Side note: this change is specifically because some projects in the OS use an OLD STL version that does not have the `std::initializer_list` constructor for `std::vector`*